.github/workflows/fortify-on-demand-scan.yml

```diff
@@ -0,0 +1,29 @@
+name: Fortify on Demand Scan
+
+# Controls when the action will run. 
+on:
+  push:
+    branches: ['master', '[0-9]?.[0-9]?.x']
+  pull_request:
+    branches: ['master', '[0-9]?.[0-9]?.x']
```

Comment:
This can just be `on: [push, pull_request]`. We're not going to have branches other than `master` and our release branches. If we do in the future then we can change it up.

Reply:
Updated